libqtile/widget/tasklist.py

```diff
@@ -141,6 +149,14 @@ class TaskList(base._Widget, base.PaddingMixin, base.MarginMixin):
             None,
             "Icon size. " "(Calculated if set to None. Icons are hidden if set to 0.)",
         ),
+        (
+            "theme_mode",
+            None,
+            "When to use theme icons. `None` = never, `preferred` = use if available, "
+            "`fallback` = use if app does not provide icon directly. "
+            "`preferred` and `fallback` have identical behaviour on Wayland.",
+        ),
+        ("theme_path", None, ""),
```

Comment:
Perhaps a doc here? Just reading through https://pyxdg.readthedocs.io/en/latest/icontheme.html and it looks like `None` is a good default, so it might be worth mentioning that `None` will use the default icon theme in the doc (if I've understood that page correctly)